packages/vue-dot/playground/examples/DialogBoxEx.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+	<DocSection title="DialogBox">
+		<h2 class="subtitle-1 mb-4 font-weight-bold">
+			Show list of button actions
+		</h2>
+
+		<VBtn
+			color="primary"
+			class="ml-6 px-5"
+			@click="dialog=true"
+		>
+			CLICK ON ME
```

Comment:
```suggestion
			Show dialog
```
On n'écrit pas en majuscules pour des raisons d'accessibilité (même si on n'est pas sur une vraie application dans le Playground, on applique quand même les bonnes pratiques)
Et surtout il faut mettre du texte qui a du sens :slightly_smiling_face:

Reply:
ça marche.nickel